lib/compat.php

```diff
@@ -169,6 +169,7 @@ function register_block_type_from_metadata( $file_or_folder, $args = array() ) {
 		$property_mappings = array(
 			'title'       => 'title',
 			'category'    => 'category',
+			'context'     => 'context',
```

Comment:
I guess we need to add `providesContext`, too :)

Reply:
fixed in #23212